### PR TITLE
Add filebeat sidecars that can be enabled

### PR DESCRIPTION
**Pull request by t-persson**
### Applicable Issues

Fixes: #70 

### Description of the Change

Add filebeat that can be enabled in the helm charts. This is disabled by default.
Not that we do not install ELK in these helm charts and it has to be started outside of ETOS to work.

### Alternate Designs

We should investigate whether we want to add ELK to ETOS as well.

### Benefits

Centralized logging is always good. This enabled the use of logstash if one has an instance up and running.

### Possible Drawbacks

Possibly locking ourselves into ELK.

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Tobias Persson <tobias.persson@axis.com>

### CODE REVIEW

**Review by andjoe-axis — APPROVED** [review]
+1

File: charts/etos/charts/etos-api/templates/deployment.yaml [hunk 1]
```diff
@@ -16,25 +16,47 @@ spec:
...
-                name: rabbitmq
-                key: password
-          - name: RABBITMQ_USERNAME
-            valueFrom:
-              secretKeyRef:
-                name: rabbitmq
-                key: username
-        ports:
-        - containerPort: 80
+        - name: {{ include "etos-api.name" . }}
```
Comment (fredjn) [inline charts/etos/charts/etos-api/templates/deployment.yaml:28]: Indentation is wrong.

File: charts/etos/charts/suite-starter/templates/deployment.yaml [hunk 2]
```diff
@@ -36,3 +44,17 @@ spec:
           envFrom:
           - configMapRef:
               name: etos
+        {{- if .Values.global.filebeatSidecar.enabled }}
```
Comment (fredjn) [inline charts/etos/charts/suite-starter/templates/deployment.yaml:47]: Indentation.

**Review by fredjn — CHANGES_REQUESTED** [review]

File: charts/etos/charts/etos-api/templates/deployment.yaml (hunk 1 above)
Comment (t-persson) [reply charts/etos/charts/etos-api/templates/deployment.yaml:28]: Indentation is correct in the generated yaml. If I indent, I cannot deploy.

File: charts/etos/charts/suite-starter/templates/deployment.yaml (hunk 2 above)
Comment (t-persson) [reply charts/etos/charts/suite-starter/templates/deployment.yaml:47]: Indentation is correct in the generated yaml. If I indent, I cannot deploy.

**Review by fredjn — APPROVED** [review]

**Pull request merged**